examples/threshold_enc.rs

```diff
@@ -97,47 +96,37 @@ struct DecryptionMeeting {
 
 impl DecryptionMeeting {
     // An actor contributes their decryption share to the decryption process.
-    fn accept_decryption_share(&mut self, actor: &mut Actor) -> Result<()> {
-        let ciphertext = actor
-            .msg_inbox
-            .take()
-            .ok_or_else(|| eyre!("no msg in inbox"))?;
+    fn accept_decryption_share(&mut self, actor: &mut Actor) {
+        let ciphertext = actor.msg_inbox.take().unwrap();
```

Comment:
We have a general policy to avoid using unwrap().   Probably we should return an Err(...) here.

If there are many unwraps and removing them would be too large an effort/change for this PR, I think it should be ok to do in a follow-up PR...